plasmapy/physics/distribution.py

```diff
@@ -0,0 +1,94 @@
+"""Functions to deal with distribution : generate, fit, calculate"""
+
+from astropy import units
+
+from astropy.units import (UnitConversionError, UnitsError, quantity_input,
+                           Quantity)
+
+from ..constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi, e)
+from ..atomic import (ion_mass, charge_state)
```

Comment:
Once again, ( ) ?

Reply:
I've made a habit of using parentheses when doing long `import` statements, as per [PEP 328](https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses).  At least to me, this makes it easier to read.  It is at its most useful when `import` statements span multiple lines.

Reply:
Oh right. I'm sorry, I forgot about this and I was checking this in a slight rush. Multilines, sure. Here, not particularly certain they're quite necessary!